frontend/tasks/content.js

```diff
@@ -0,0 +1,150 @@
+    .pipe(buildExperimentsJSON())
+    .pipe(gulp.dest(config.DEST_PATH + 'api'));
+});
+
+gulp.task('import-api-content', (done) => {
+  fetch(config.PRODUCTION_EXPERIMENTS_URL)
+    .then(response => response.json())
+    .then(data => Promise.all(data.results.map(processImportedExperiment)))
+    .then(() => done())
+    .catch(done);
+});
+
+function buildExperimentsJSON() {
+  const index = {results: []};
+  const counts = {};
+
+  function collectEntry(file, enc, cb) {
+    const yamlData = file.contents.toString();
+    const experiment = YAML.parse(yamlData);
+
+    // Auto-generate some derivative API values expected by the frontend.
+    Object.assign(experiment, {
+      url: `/api/experiments/${experiment.id}.json`,
+      html_url: `/experiments/${experiment.slug}`,
+      installations_url: `/api/experiments/${experiment.id}/installations/`,
+      survey_url: `https://qsurvey.mozilla.com/s3/${experiment.slug}`
+    });
+
+    counts[experiment.addon_id] = experiment.installation_count;
+    delete experiment.installation_count;
+
+    this.push(new gutil.File({
+      path: `experiments/${experiment.id}.json`,
+      contents: new Buffer(JSON.stringify(experiment, null, 2))
+    }));
+
+    index.results.push(experiment);
+
+    cb();
+  }
+
+  function endStream(cb) {
+    this.push(new gutil.File({
+      path: 'experiments.json',
+      contents: new Buffer(JSON.stringify(index, null, 2))
+    }));
+    this.push(new gutil.File({
+      path: 'experiments/usage_counts.json',
+      contents: new Buffer(JSON.stringify(counts, null, 2))
+    }));
+    cb();
+  }
+
+  return through.obj(collectEntry, endStream);
+}
+
+function processImportedExperiment(experiment) {
+  // Clean up auto-generated and unused model fields.
+  const fieldsToDelete = {
+    '': ['url', 'html_url', 'installations_url', 'survey_url'],
+    details: ['order', 'url', 'experiment_url'],
+    tour_steps: ['order', 'experiment_url'],
+    contributors: ['username']
+  };
+  Object.keys(fieldsToDelete).forEach(key => {
+    const items = (key === '') ? [experiment] : experiment[key];
+    const fields = fieldsToDelete[key];
+    items.forEach(item => fields.forEach(field => delete item[field]));
+  });
+
+  // Download all the images associated with the experiment.
+  const imageFields = {
+    '': ['thumbnail', 'image_twitter', 'image_facebook'],
+    details: ['image'],
+    tour_steps: ['image'],
+    contributors: ['avatar']
+  };
+  const toDownload = [];
+  Object.keys(imageFields).forEach(key => {
+    const items = (key === '') ? [experiment] : experiment[key];
+    const fields = imageFields[key];
+    items.forEach(item => fields.forEach(field => {
+      // Grab the original image URL, bail if it's not available
+      const origURL = item[field];
+      if (!origURL) { return; }
+
+      // Chop off the protocol & domain, convert gravatar param to .jpg
+      const path = origURL.split('/').slice(3).join('/').replace('?s=64', '.jpg');
```

Comment:
Probably well out of scope of this PR, but I had to read that `.split().slice(3).join()` a couple times to figure it out. Not sure if it's easier to use `url.parse().path` instead (only difference seems to be the leading "/"):

```js
const url = require('url');

const origURL = "https://msn.com/foo/bar/baz/image?s=64";

const path = origURL.split('/').slice(3).join('/').replace('?s=64', '.jpg');
const path2 = url.parse(origURL).path.replace('?s=64', '.jpg');

console.log(path);  // "foo/bar/baz/image.jpg"
console.log(path2); // "/foo/bar/baz/image.jpg"
```